src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceCreate.kt

```diff
@@ -27,53 +23,13 @@ open class InstanceCreate : LocalInstanceTask() {
 
         logger.info("Creating instances: ${uncreatedInstances.names}")
 
-        val backupZip = when (instanceOptions.source) {
-            LocalInstanceOptions.Source.AUTO -> findRecentBackup()
-            LocalInstanceOptions.Source.BACKUP_INTERNAL -> getInternalBackup()
-            LocalInstanceOptions.Source.BACKUP_EXTERNAL -> getExternalBackup()
-            LocalInstanceOptions.Source.NONE -> null
-        }
-        if (backupZip != null) {
-            createFromBackup(uncreatedInstances, backupZip)
-        } else {
-            createFromScratch(uncreatedInstances)
-        }
+        createFromScratch(uncreatedInstances)
```

Comment:
`fromScratch` now have no sense / nothing need to be distinguished

Reply:
OK